src/sentry/utils/snuba.py

```diff
@@ -33,8 +35,12 @@ def query(start, end, groupby, conditions=None, filter_keys=None,
     `aggregations` a list of (aggregation_function, column, alias) tuples to be
     passed to the query.
     """
+    start = start.replace(tzinfo=pytz.utc) if not start.tzinfo else start
+    end = end.replace(tzinfo=pytz.utc) if not end.tzinfo else end
```

Comment:
Same comment as above about tracking down where these come from and fixing them, we should try and avoid this if we can.